### DISCUSSION TITLE
Change notification level based on spaces

### DISCUSSION THREAD

**Discussion by Cadair** [element-hq/element-meta#1306] [Ideas]
It would be a nice feature if you could change the notification level of all rooms in a community.

I am thinking of a use case, where a user has one matrix account both used for work and for personal purposes, and all the work rooms are in one or two communities. It would be really helpful for the sanity of this imaginary user if they could mute the work related rooms over the weekend / while on holiday.

**Comment by MurzNN** [comment]
Maybe better is to use room tags for notification tune-up? Eg create room tag 'Work' and move all work chats to it, configure notification rule for this tag. Tags are private for users, but most of communities are shared and not always controlled by you.

**Comment by Cadair** [comment]
I am not suggesting that changing this notification level would change it for *all* users in a community, just as a shortcut for you to batch change all the notification levels for all the rooms you are joined in on your account. (The same effect as you doing them 1-1 with the current Riot interface).

**Comment by daniellekirkwood** [comment]
This is definitely something we've been considering - thanks for the suggestion. I'm going to move this issue over to the discussions section so it's referenced and handled correctly as a product improvement suggestion.
